LinkedListProject/Node.cpp

```diff
@@ -4,10 +4,15 @@ using namespace std;
 
 Node::Node() {}
 
-Node::Node(int iData) {
-	this->data = iData;
+Node::Node(int data) {
+	this->data = data;
 }
+
+//Can't catch deconstructor of Node
+Node::~Node() {
+	cout << "Catch descontructor of node";
+}
+
 void Node::DislayNode() {
```

Comment:
this one is okay, but I want you to override << operator, so that we can do this:
cout << node << endl;